lib/matplotlib/tests/test_streamplot.py

```diff
@@ -60,9 +54,9 @@ def test_linewidth():
                    linewidth=lw)
 
 
+# Tolerance due to change in sticky-edges logic.
 @image_comparison(baseline_images=['streamplot_masks_and_nans'],
-                  tol=0.04 if on_win else 0,
-                  remove_text=True, style='mpl20')
+                  tol=.1, remove_text=True, style='mpl20')
```

Comment:
Why bump the tolerance instead of replacing the test image?

Reply:
see https://gitter.im/matplotlib/matplotlib?at=5c87e610ac408e11924acaa9

Reply:
We definitely shouldn't bump tolerances because the actual test image has changed, as this means any small changes to the images in future will not be picked up.

Reply:
Sure, I just forced the axes limits to their old values instead...